src/MathJaxRenderer.jl

```diff
@@ -28,10 +28,10 @@ either `show` or `write`:
       - `dpi_y = 90`, pixels per inch (vertical).
       - `x_zoom = 1.0`, zoom factor (horizontal).
       - `y_zoom = 1.0`, zoom factor (vertical).
-      - `zoom = 1.0`, zoom factor (both horizontal and vertical).
```

Comment:
Probably worth keeping `zoom` as part of the API that this package provides otherwise it's kind of a breaking change to just remove it. What could be done is allow `zoom` and just pass it's value to `x_zoom` and `y_zoom` internally.

Reply:
I did not address this because `rsvg-convert` also broke the API -- feel free to add it yourself :)